### PR TITLE
#57 Remove Dgraph GraphQL layer

### LINKED ISSUE #57: Remove Dgraph GraphQL references from code/docs
Initial approach using Dgraph GraphQL uncovered a lot of GraphQL features still in beta. Approach is now to go from client -> Apollo GraphGL -> Dgraph GraphQL+- 

Doc/code cleanup is needed to reflect this change.

## Expected Behavior

- Remove Dgraph GraphQL schema
- Revert scripts to setup Dgraph directly (rather than GraphQL admin/schema endpoint).

### PR DESCRIPTION
**Pull request by katherine-sussol**
Fixes #57

## Description

Undo Dgraph GraphQL schema changes introduced for #29, as it was replaced by #52.

### Documentation:

Tick below if one of the following applies:

1. Documentation has been added to cover changes introduced by this PR.
2. This PR requires no changes to documentation.

- [x] Documentation OK.

### Tests:

Tick below if one of the following applies:

1. Tests have been added to cover changes introduced by this PR.
2. This PR requires no new tests.

- [x] Tests OK.

### CODE REVIEW

File: dgraph/sample/schema.gql [hunk 1]
```diff
@@ -1,13 +1,20 @@
-type Entity @dgraph(type: "Entity") {
-    type: String! @dgraph(pred:"type") @search(by: [exact])
-    code: String @dgraph(pred:"code") @search(by: [exact])
-    description: String! @dgraph(pred:"description") @search(by: [fulltext, exact])
-    properties: [Property] @dgraph(pred:"has_property")
-    children: [Entity] @dgraph(pred:"has_child")
+type Entity {
```
Comment (wlthomson) [inline dgraph/sample/schema.gql:1]: Is this a duplicate of the Apollo schema?

Comment (katherine-sussol) [reply dgraph/sample/schema.gql:1]: Yeah, they are pretty close (at the moment the Entity and Property nodes are 1-1 directly mapped), but I think we still need it to create the Dgraph schema when setting up the DB? 

The alternative is to use a single schema file to set up both Dgraph and Apollo, then Apollo will need a separate script/routine to add in the additional fields we need (for interactions/custom queries etc).

Comment (wlthomson) [reply dgraph/sample/schema.gql:1]: My mistake, thought this was the old GraphQL schema (not the Dgraph schema)!

I think we can probably move this out of the `sample/` dir now (but can do that in another PR maybe).

Comment (katherine-sussol) [reply dgraph/sample/schema.gql:1]: That's a good point - it's not really a sample anymore is it? :D

**Pull request merged**